### PR TITLE
Fix oss-fuzz 42193 [13449]

### PR DESCRIPTION
**Pull request by MiguelCompany**
Fixed a null-dereference on XML parser

### CODE REVIEW

File: test/unittest/xmlparser/regressions/13418.xml [hunk 1]
```diff
@@ -0,0 +1 @@
+<types><type><typedef type=""arrayDimensions=""/></type></types>
```
Comment (EduPonz) [inline test/unittest/xmlparser/regressions/13418.xml:1]:
```suggestion
<types><type><typedef type=""arrayDimensions=""/></type></types>

```

**Review by EduPonz — CHANGES_REQUESTED** [review]
NITPICK: I've added the `skip-ci` label as the changes look good to me

File: test/unittest/xmlparser/regressions/13418.xml (hunk 1 above)
Comment (MiguelCompany) [reply test/unittest/xmlparser/regressions/13418.xml:1]: I'm not going to take this, as the regression file is the one downloaded from the oss-fuzz report.

**Review by EduPonz — APPROVED** [review]

**Comment by MiguelCompany** [comment]
@mergifyio backport 2.3.x

**Pull request merged**